lib/active_fedora/relation/finder_methods.rb

```diff
@@ -189,17 +189,16 @@ def find_one(id, cast = nil)
     protected
 
       def load_from_fedora(id, cast)
-        raise ActiveFedora::ObjectNotFoundError if id.empty?
+        if id.empty?
+          raise ActiveFedora::ObjectNotFoundError, "No ID provided for #{klass.name}."
+        end
         resource = ActiveFedora.fedora.ldp_resource_service.build(klass, id)
-        raise_record_not_found_exception!(id) if resource.new?
+        if resource.new?
```

Comment:
We should favor the if/unless modifer style per https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier